src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -1069,7 +1069,23 @@ GlobeControls.prototype.isAnimationEnabled = function isAnimationEnabled() {
  * @return     {number}  The zoom .
  */
 GlobeControls.prototype.getZoom = function getZoom() {
-    return computeTileZoomFromDistanceCamera(this.getRange(), this._view);
+    const distance = this.getRange();
```

Comment:
I prefer move `computeTileZoomFromDistanceCamera` in GlobeLayer with this parameters:

`computeTileZoomFromDistanceCamera(distance, camera)`

Reply:
Done